HLTrigger/special/src/HLTL1NumberFilter.cc

```diff
@@ -76,6 +84,10 @@ HLTL1NumberFilter::filter(edm::StreamID, edm::Event& iEvent, const edm::EventSet
     if (data.data() && data.size() > 0){
       FEDHeader header(data.data()) ;
       if (period_!=0) accept = ( ( (header.lvl1ID())%period_ ) == 0 );
+      if (useTCDS_ && period_!=0) {
+	evf::evtn::TCDSRecord record((unsigned char *)data.data());
+	accept = ( (record.getHeader().getData().header.triggerCount % period_) == 0 );
+      }
```

Comment:
Actually, one could move the `if (period_!=0)` even before the access to the event, though it is quite unlikely for this filter to be used that way.